flattentool/input.py

```diff
@@ -105,7 +112,11 @@ def convert_type(type_string, value, timezone=pytz.timezone("UTC"), convert_flag
         if type(value) == datetime.datetime:
             return value.date().isoformat()
         return str(value)
-    elif convert_flags.get("wkt") and type_string == "geojson":
+    elif (
+        convert_flags.get("wkt")
+        and type_string == "geojson"
+        and SHAPELY_AND_GEOJSON_LIBRARIES_AVAILABLE
```

Comment:
Can we have a warning if the first 2 are true, but the 3rd isn't? (And similarly in `json_input.py`).

Reply:
That's even worse to write tests for :-P Good point

Reply:
Seen in tests:

```
==================================================================================================================================== warnings summary ====================================================================================================================================
flattentool/tests/test_json_input.py::test_parse_geojson
  .../flattentool/json_input.py:382: UserWarning: Install flattentool's optional geo dependencies to use geo features.
    warn(

```